chatgpt.js

```diff
@@ -1515,20 +1516,19 @@ const chatgpt = { // eslint-disable-line no-redeclare
     reviewCode() { chatgpt.code.review(); },
     scrollToBottom() { try { chatgpt.getScrollBtn().click(); } catch (err) { console.error(err.message); }},
 
-    send(msg, method='') {
-        for (let i = 0; i < arguments.length; i++) if (typeof arguments[i] !== 'string')
-            return console.error(`Argument ${ i + 1 } must be a string!`);
+    async send(msg, method='') {
+        if (typeof msg !== 'string') return console.error('Message must be a string!');
         const textArea = chatgpt.getChatBox();
         if (!textArea) return console.error('Chatbar element not found!');
-        const msgP = document.createElement('p'); msgP.textContent = msg;
+        const msgP = document.createElement('p'); 
+        msgP.textContent = msg;
```

Comment:
It saved space on one line and pertained to `msgP` init so was logically grouped